docs/content/latest/explore/ysql-language-features/advanced-features/partitions.md

```diff
@@ -99,7 +99,39 @@ To create a new partition that contains only the rows that don't match the speci
 CREATE TABLE order_changes_default PARTITION OF order_changes DEFAULT;
 ```
 
-Optionally, you can create an index on the key columns and other indexes for every partition, as follows:
+Optionally, you can create indexes on a partitioned table.
```

Comment:
Suggestion:

Optionally, you can create indexes on a partitioned table by executing the following command: